### ISSUE TITLE
Getting the following certificate error for a brand new kube-aws cluster

### ISSUE THREAD

**Issue by gordonmurray** [coreos/coreos-kubernetes#766]
I get the following error when running kubectl cluster-info on a new kubernetes cluster created using kube-aws.

**Unable to connect to the server: x509: certificate signed by unknown authority**

I followed this guide on a new EC2 instance:
https://coreos.com/kubernetes/docs/latest/kubernetes-on-aws.html

I followed this guide to install kubectl
https://coreos.com/kubernetes/docs/latest/configure-kubectl.html

These guides worked for me a few days ago, however my recent attempts, starting from a clean slate result in this error.

My cluster.yaml file is using the default settings, accept for changing the m3.medium to m4.large instances inside a new VPC which kube-aws creates.

Has anyone else had this error, or would know how to debug it?

Thank you.

**Comment by robszumski** [comment]
Are you seeing that with `kubectl <command>`? Did you swap out your old `.kubeconfig` for the new one?

**Comment by gordonmurray** [comment]
Yes, I see it when I try to use : kubectl cluster-info or kubectl get nodes

I don't have any old config, I'm starting from a clean slate each time.

**Comment by pieterlange** [comment]
Can you try this with the [new kube-aws](https://github.com/coreos/kube-aws/releases/tag/v0.9.1)?

**Comment by gordonmurray** [comment]
Ill give it a go

**Comment by fastestturtle** [comment]
Hey @gordonmurray , did you resolve your issue? it looks like I am having the same issue building a brand new cluster.
